Applications/LoRATrainingExample/ContentView.swift

```diff
@@ -150,7 +150,6 @@ class LoRAEvaluator {
                         limit: 1.0)
                 }
             }
-            eval(model)
```

Comment:
This shouldn't have been there -- the load of the model already eval'd it and this model is actually an enum so it hits an assert, see #102.